modules/accessing-metrics-outside-cluster.adoc

```diff
@@ -54,8 +54,35 @@ $ curl -H "Authorization: Bearer $TOKEN" -k "https://$HOST/api/v1/query?" --data
 +
 The output shows the status for each application pod that Prometheus is scraping:
 +
-.Example output
+.The formatted example output
 [source,terminal]
 ----
-{"status":"success","data":{"resultType":"vector","result":[{"metric":{"__name__":"up","endpoint":"web","instance":"10.129.0.46:8080","job":"prometheus-example-app","namespace":"ns1","pod":"prometheus-example-app-68d47c4fb6-jztp2","service":"prometheus-example-app"},"value":[1591881154.748,"1"]}]}}
+{
+  "status": "success",
+  "data": {
+    "resultType": "vector",
+    "result": [
+      {
+        "metric": {
+          "__name__": "up",
+          "endpoint": "web",
+          "instance": "10.129.0.46:8080",
+          "job": "prometheus-example-app",
+          "namespace": "ns1",
+          "pod": "prometheus-example-app-68d47c4fb6-jztp2",
+          "service": "prometheus-example-app"
+        },
+        "value": [
+          1591881154.748,
+          "1"
+        ]
+      }
+    ],
+  }
+}
```

Comment:
Outside the scope of this PR. But, would the user benefit from an explanation of what he is seeing in the example?

Reply:
Hmm, I wonder, I personally do not think there is anything that would benefit from an additional explanation :thinking: . 

@simonpasquier do you think there is something in the output that would be useful for users to point out/explain more?

(I will not hold this PR, but I will create an additional issue for this if the answer is yes)

Reply:
Maybe we should mention that the command requests the "Instant Query" endpoint of the Thanos Querier service?

Reply:
Because I do not want to hold this PR before the releae deadline, I created a Jira for this: https://issues.redhat.com/browse/OBSDOCS-1333
as I need more information regarding this.  Thanks!